crates/sui-bridge-indexer/src/metrics.rs

```diff
@@ -18,9 +18,8 @@ pub struct BridgeIndexerMetrics {
     pub(crate) total_eth_token_transfer_claimed: IntCounter,
     pub(crate) total_eth_bridge_txn_other: IntCounter,
     pub(crate) last_committed_sui_checkpoint: IntGauge,
-    pub(crate) last_committed_eth_block: IntGauge,
-    pub(crate) last_synced_unfinalized_eth_block: IntGauge,
-    pub(crate) last_committed_unfinalized_eth_block: IntGauge,
```

Comment:
I think we still want these for the keep-up task

Reply:
True I deleted these when reworking syncing, but now that I'm adding in subscriptions, we can add these back in.